lms/views/api/canvas/pages.py

```diff
@@ -60,7 +129,8 @@ def via_url(self):
                 self.request.route_url(
                     "canvas_api.pages.proxy",
                     _query={
-                        "document_url": assignment.document_url,
+                        "course_id": current_course_id,
```

Comment:
The proxy endpoints changes, from being based on the document_id to be explicitly told which course and page ids to use.